.github/workflows/Java-Instrumentation-Tests.yml

```diff
@@ -50,6 +50,9 @@ jobs:
       - name: Build newrelicJar
         run: ./gradlew $GRADLE_OPTIONS clean jar
 
+      - name: Remove parallel from GRADLE_OPTIONS
+        run: echo "GRADLE_OPTIONS=$GRADLE_OPTIONS" | sed "s/--parallel//" >> $GITHUB_ENV
+
```

Comment:
This prevents the integration tests from running in parallel, thus preventing any 2 of  them from using the same resource at the same time.